src/Mvc/Directory.Build.props

```diff
@@ -1,7 +1,7 @@
 <Project>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory)..\, Directory.Build.props))\Directory.Build.props" />
 
-  <PropertyGroup Condition=" '$(IsTestProject)' == 'true' ">
-    <CodeAnalysisRuleSet>$(MSBuildThisFileDirectory)MvcTests.ruleset</CodeAnalysisRuleSet>
-  </PropertyGroup>
+  <ItemGroup>
+    <EditorConfigFiles Include="$(MSBuildThisFileDirectory)MvcTests.globalconfig" />
```

Comment:
@sharwell I'm still getting warnings from a disabled analyzer. Can you give a look?